discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+interaction with your website pages and assets is a bigger part. This type of
+analytics (called Touchpoint Analytics) helps you determine content
+effectiveness in achieving your business goals. 
+
+## Touchpoint Interaction
+
+*Touchpoints* are the locations (pages) where people interact with content. Here
+are some of the Touchpoint Analytics: 
+
+-   Visitor count: the number of users (guest and signed in) visiting a 
+    Touchpoint
+-   Session length: amount of time people are spending on a channel (e.g., 
+    Liferay DE instance). 
+-   Number of sessions: number of different people at a Touchpoint
+-   Time on page: amount of time people are spending at a Touchpoint
+-   Bounce rate: percentage of visitors to a particular website who navigate 
+    away from the site after viewing only one Touchpoint
+
+*Assets* are specific types of content entities on Touchpoints. Traditional web 
+analytics are based on static websites---everyone sees the same page content. By
+additionally providing analytics on content articles, Analytics Cloud helps you
+determine the effectiveness of those articles. Asset Analytics also provides
+insight into people's interaction with dynamic personalized content. It
+generates charts on target audiences engage the content.
+
+Liferay Analytics Cloud release 1.0 supports these Asset types from Liferay
+Digital Enterprise:
+
+- Blogs
+- Documents and Media
+- Forms
+
+Regarding user behavior with a form, for example, Analytics Cloud shows the
+number of views and submissions, the percentage of users abandoning the form,
+and the form's average completion time. A line graph shows the number of views
+over time.
+
+![Figure 1: Analytics Cloud shows you how users interact with your forms.](../images/intro-form-behavior-card.png)
+
+As you view Touchpoint and Asset Analytics you can discern whether an action is
+positive or negative. Here are some examples. 
+
+Positive:
+
+- staying active in sessions
+- frequent sign in
+- submitting a form 
+- giving an article five stars
+
+Negative:
+
+- signing out shortly after signing in
+- passing by key strategic content
+- abandoning a form
+- letting a session timeout
+
+The analytics help you determine better ways to relate to people and earn their
+business. For example, if the majority of people in a target industry Segment
+reactive positively to content that describes a new service you're selling, the
+Segment might be interested in similar services. 
+
+Analytics Cloud uncovers issues too. If lots of people abandon a form after
+entering data leading up to a particular field, for example, the field might be
+problematic. If the field's label is ambiguous, make it clear. If filling in the
+field is cumbersome, make it multiple choice. If the field is more trouble than
+its worth, remove it. The point is Analytics Cloud uncovers trouble spots for
+you to address. Identifying and resolving issues makes your websites better and
+helps you achieve business goals. 
```

Comment:
The information in the above two paragraphs might be good for the top intro section, to introduce people to the benefits of analytics. Not sure it should wait till after you've described some of the technical details about individuals, segments, and touchpoints. You might want to mention the examples at the beginning and then refer back to them here.